src/MinkowskiSum.jl

```diff
@@ -343,10 +368,65 @@ function σ(d::AbstractVector{<:Real}, cms::CacheMinkowskiSum)::Vector{<:Real}
         # first-time computation of support vector
         svec = _σ_helper(d, arr)
     end
-    cache[d] = (l, svec)
+    cache[d] = CachedPair(l, svec)
     return svec
 end
 
+"""
+    forget_sets!(cms::CacheMinkowskiSum)::Int
+
+Tell a caching Minkowski sum to forget the stored sets.
```

Comment:
... without actually deleting the cache, if i understand correctly?

Reply:
Yep, of course :grin: